src/components/LanguageSwitch.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Select from "@material-ui/core/Select";
+import MenuItem from "@material-ui/core/MenuItem";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import { useIntlContext } from 'src/translations/IntlContext';
+
+const LanguageSwitch = () => {
+  const { locale, localeKeys, switchToLanguage } = useIntlContext()
```

Comment:
check your naming - you have aliases in your code locale - lang (choose one of them)